Directory.Build.props

```diff
@@ -29,6 +29,10 @@
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'">
+    <IsTrimmable>true</IsTrimmable>
+  </PropertyGroup>
```

Comment:
Do we also want to try 
```xml
<EnableMicrosoftExtensionsConfigurationBinderSourceGenerator>true</EnableMicrosoftExtensionsConfigurationBinderSourceGenerator>
```
if we use [Microsoft.Extensions.Configuration.Binder NuGet package](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.Binder) 8.*?

and then remove the attributes which are due to the binding.

Shall we do that when we enable .NET 8?
@jennyf19 @westin-m ?

Reply:
Let's do that as an increment.

Reply:
Yes, we should do this with .NET 8.

Reply:
Yes, I want to try this with .NET 8.